raiden/storage/sqlite.py

```diff
@@ -27,6 +28,18 @@
 )
 
 
+class Operator(Enum):
+    NONE = ""
+    AND = "AND"
+    OR = "OR"
+
+
+class Query(NamedTuple):
```

Comment:
Also add a docstring here. And perhaps rename to `FilteredDBQuery` as `Query` is way too generic a name.